specification/validated-relationships-service-api.yaml

```diff
@@ -377,20 +502,26 @@ components:
                               properties:
                                 system:
                                   type: string
-                                  enum: ['http://terminology.hl7.org/CodeSystem/v3-RoleCode']
+                                  enum:
+                                    [
+                                      "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+                                      "http://terminology.hl7.org/CodeSystem/v3-RoleClass",
+                                      "https://fhir.hl7.org.uk/CodeSystem/UKCore-AdditionalPersonRelationshipRole"
+                                    ]                                
                                 code:
                                   type: string
-                                  enum: ['PRN', 'Personal']
+                                  enum: ['PRN', 'CHILD', 'SPS', 'FAMMEMB', 'FRND', 'CAREGIVER', 'SIGOTHR']
                                 display:
                                   type: string
-                                  example: "Personal"
-                                  
-    QuestionnaireItem_PatientDetails:
+                                  description: FHIR relationship type.
+                                  enum: ['Parent', 'Child', 'Partner, wife or husband', 'Other family member', 'Friend, neighbour or housemate', 'Paid carer or support worker', 'Other']
+
+    QuestionnaireItem_ParentalApplicationSupplementaryDetails:
       type: object
       properties:
         linkId:
           type: string
-          enum: ["patient_details"]
+          enum: ["parentalApplicationSupplementaryDetails"]
         text:
```

Comment:
Suggest adding an example "Supplementary details about a parental application" for parity with most other fields, currently this one comes out as `"text": "string"` in the generated swagger docs.